sgt-back-end/index.js

```diff
@@ -0,0 +1,143 @@
+const express = require('express');
+const app = express();
+app.use(express.json());
+const pg = require('pg');
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+app.get('/api/grades', (req, res) => {
+  const sql = `
+    select *
+      from "grades"
+  `;
+  db.query(sql)
+    .then(result => {
+      res.status(200).json(result.rows);
+    })
+    .catch(err => {
+      console.error(err);
+      res.status(500).json({
+        error: 'An unexpected error ocurred.'
+      });
+    });
+});
+
+app.post('/api/grades', (req, res) => {
+  const score = parseFloat(req.body.score);
+
```

Comment:
Since you are using values from the `body` in several places it is best to first save them into variables then use those variables in the rest of the function. For example:
```JAVASCRIPT
const name = req.body.name;
const course = req.body.course;
const score = parseFloat(req.body.score);

// -OR- Use ES6 Desturcturing
const { name, course } = req.body;
const score = parseFloat(req.body.score);
```